readme.md

```diff
@@ -6,11 +6,13 @@ Project Apicula uses a combination of fuzzing and parsing of the vendor data fil
 
 ##  Dependencies
 
-Version 1.9 of the Gowin vendor tools, and a copy of the Excel pinout files downloaded in `~/Documents/gowinsemi`
+Version 1.9.1.01 of the Gowin vendor tools, and a copy of the Excel pinout files downloaded in `~/Documents/gowinsemi`:
+* UG107-1.07E_GW1N-1 Pinout.xlsx
+* UG801-1.5E_GW1NR-9 Pinout.xlsx
 
 The latest Yosys and Nextpnr, installed with the generic backend.
 
-Python 3.7+:
+Python 3.8:
```

Comment:
this is no longer true, should work on 3.6 now with the dataclasses module

Reply:
I thought the start parameter in sum() was new in 3.8.

Reply:
Yes, so I'm no longer using it, see 2d8d2e7653bfbe33876f51f13603dd137990ae8d